### ISSUE TITLE
"it" variable in JS

### ISSUE THREAD

**Issue by faiwer** [borela/naomi#50]
![img_18_01_12 24 38_df7e5e1d943](https://user-images.githubusercontent.com/744114/35083702-9343f0a8-fc4a-11e7-84a1-b06adbb74a4c.png)
```js
for(const it of el.values)
	if(it.length)
		mea.eval(it); // throw
return el.values.length ? el.values[0] : '1';
```
Hello. I don't know why, but the code abode breaks all syntax highlighting below :) My expreriments with it shows that it because I use here the variable called `it`. When I changed it to `a` the trouble dissolved.

**Comment by borela** [comment]
Fix https://github.com/borela/naomi/releases/tag/v3.1.2 It was highlighting the jest global as it has `it.skip`, `it.only`.

**Comment by borela** [comment]
Fixed on >= 3.1.2
